pkg/clusterd/context.go

```diff
@@ -66,4 +67,7 @@ type Context struct {
 
 	// The local devices detected on the node
 	Devices []*sys.LocalDisk
+
+	// OrchestrationCanceller manages the orchestration and its possible cancellation
+	OrchestrationCanceller *abool.AtomicBool
```

Comment:
As a nit, the methods on this are `Set`, `Unset`, and `IsSet`, and I don't read "(un)set canceller" as having the right semantic meaning. I think renaming this to "RequestCancelOrchestration" or "OrchestrationCancellationRequest" or something similar makes the method use more grammatically specific to what's going on in the code.

Reply:
Will go with `RequestCancelOrchestration` thanks for the suggestion.